File: fizzbuzz.clj
(ns fizzbuzz)

(defn fizzbuzz [n]
  (cond
    (zero? (mod n 15)) "FizzBuzz"
    (zero? (mod n 3)) "Fizz"
    (zero? (mod n 5)) "Buzz"
    :else (str n)))

(defn -main []
  (doseq [n (range 1 101)]
    (println (fizzbuzz n))))

File: fizzbuzz.nim
proc fizzbuzz(n: int): string =
  if n mod 15 == 0:
    return "FizzBuzz"
  elif n mod 3 == 0:
    return "Fizz"
  elif n mod 5 == 0:
    return "Buzz"
  else:
    return $n

for i in 1..100:
  echo fizzbuzz(i)
